packages/openapi-generator/src/file-serializer/api-file.ts

```diff
@@ -1,23 +1,30 @@
+import { EOL } from 'os';
 import { codeBlock, unique } from '@sap-cloud-sdk/util';
-import { OpenApiApi, OpenApiOperation } from '../openapi-types';
+import {
+  OpenApiApi,
+  OpenApiDocument,
+  OpenApiOperation
+} from '../openapi-types';
 import {
   collectRefs,
   hasNotSchema,
   parseTypeNameFromRef
 } from '../schema-util';
 import { serializeOperation } from './operation';
 import { Import, serializeImports } from './imports';
+import { apiDocumentation } from './docs';
 
 /**
  * Serialize an API representation to a string representing the resulting API file.
  * @param api Represenation of an API.
+ * @param document Representation of the Document used the generated documentation.
  * @returns The serialized API file contents.
  */
-export function apiFile(api: OpenApiApi): string {
+export function apiFile(api: OpenApiApi, document?: OpenApiDocument): string {
```

Comment:
I thought I already commented this... 🤔

1. Why is this optional?
2. Maybe we should just pass the service name instead of the whole document?